baskup

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
No need to rename this file. `.sh` is a fine extension for bash scripts. The advantage of not renaming it is that GitHub will show us your changes inline.